web-ui/src/main/resources/catalog/components/viewer/ViewerDirective.js

```diff
@@ -71,26 +71,47 @@
               };
               scope.ol3d = null;
 
+
               // 3D mode is allowed and disabled by default
               scope.is3DModeAllowed = gnConfig['map.is3DModeAllowed'] || false;
               scope.is3dEnabled = gnConfig['is3dEnabled'] || false;
 
-
               // Synch only background layer between main map and search map
               scope.synAllLayers = false;
+
               scope.map.getLayers().on('add', function() {
                 if (angular.isDefined(gnSearchSettings.searchMap)) {
+
                   var layers = gnSearchSettings.searchMap.getLayers();
+
                   scope.map.getLayers().forEach(function(l) {
                     if (scope.synAllLayers === false &&
                         l.get('group') === 'Background layers') {
                       layers.removeAt(0);
                       layers.insertAt(0, l);
                     }
                     // TODO: Synchronize all layers
+
                   });
+
                 }
               });
+
+              scope.syncMod = function(map) {
+
+                scope.synAllLayers=!scope.synAllLayers;
+
+                var layers = gnSearchSettings.searchMap.getLayers();
+                layers.clear();
```

Comment:
I've not tested it live, but this clear will dropped all layers including the layers used for displaying bboxes ? eg https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/search/resultsview/ResultsviewDirective.js#L103